props/saber_caiwyn_buttons.h

```diff
@@ -0,0 +1,604 @@
+// Caiwyn's Lightsaber Controls
+// This config is designed to provide fast, low-latency response to button presses for the most basic functions (blaster blocks, lockups, and clashes) by omitting, or limiting non-critical features.
+// At least 2 buttons are required.  A third button can be added, but is not necessary.
+//
+// While saber is DEACTIVATED:
+// Start/Stop Track: Click Aux
+// Track Player Mode: Double-click Aux
+//   This cycles through three playback modes:
+//   1. Play a single track and stop
+//   2. Repeat a single track in a loop
+//   3. Repeat all tracks in a loop
+//   Tracks must be in <font>/tracks/*.wav, and will be selected in alphabetical order.
+// Next Track: Hold Aux while track is playing
+// Next Preset: Hold Aux when track is not playing
+// Check Battery: Double-click and hold Aux
+// Activate Saber: Press Power
+// Activate Saber and Start Track: Hold Aux or Aux2 and press Power
+//
+// While saber is ACTIVATED:
+// Non-impact Clash/Lockup: Click Power (This generates a clash/lockup effect with no impact to the blade; quick press for a short clash, hold for a lockup)
+// Blaster Block: Click Aux
+// Lightning Block: Hold Aux
+// Lockup: Hold Aux during impact
+// Drag: Hold Aux during impact with saber pointed down.
+// Melt: Hold Aux and stab
+// Enter Volume Menu: Double-click and hold Aux (Be aware that the first click will trigger a blaster block)
+//   Increase Volume: Rotate hilt right while in Volume Menu
+//   Decrease Volume: Rotate hilt left while in Volume Menu
+//   Save Changes and Exit Volume Menu: Click Power
+//   Discard Changes and Exit Volume Menu: Click Aux
+// Enter Color Change Mode (Color Wheel): Triple-click and hold Aux (Be aware that the first two clicks will trigger blaster blocks)
+//   Change Color: Rotate hilt while in Color Change Mode
+//   Color Zoom: Press & Hold Power and Rotate hilt while in Color Change Mode
+//   Save Changes and Exit Color Change Mode: Release Power
+//   Reset Color and Exit Color Change mode: Click Aux
+// Deactivate Saber: Hold Aux or Aux2 and press Power
+//
+// You will need the following sound files in order for menus to work properly:
+// vmbegin.wav - Enter Volume Change Menu
+// vmend01.wav - Save Volume Change
+// vmend02.wav - Cancel Volume Change
+// monce.wav - Set Track Player to play a single track one time
+// mloop.wav - Set Track Player to repeat a single track
+// mrotate.wav - Set Track Player to repeat all tracks
+// ccbegin.wav - Enter Color Change Mode
+// ccend01.wav - Save Color and Exit Color Change Mode
+// ccend02.wav - Reset Color and Exit Color Change Mode
+//
+// Options you can add to your config file:
+// #define DISABLE_BEEPS        - Disables the single beep that occurs when selecting fonts, tracks, or entering or exiting the volume and color change menus.
```

Comment:
All defines must have a prefix unique to your config file, something like:

CAIWYN_DISABLE_BEEPS

Reply:
Will do.

Reply:
Done.